components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -242,8 +213,8 @@ private void logFetchResults(List<BuildValidationData> buildValidationData, Cust
         }
     }
 
-    private void logFetchResultFor(int index, String property, String customValueKey, boolean found) {
-        String ordinal = runNumToOrdinal(index);
+    private void logFetchResultFor(int runNum, String property, String customValueKey, boolean found) {
+        String ordinal = toOrdinal(runNum);
```

Comment:
I would inline this local variable with 1 usage.

Reply:
✅ Done